openslides_backend/presenter/get_user_related_models.py

```diff
@@ -117,7 +117,12 @@ def get_committees_data(self, user_id: int) -> List[Dict[str, Any]]:
         }
         for level in user.get("committee_$_management_level", []):
             for committee_nr in user.get(f"committee_${level}_management_level", []):
-                committees[committee_nr]["cml"].append(level)
+                if committee_nr in committees:
+                    committees[committee_nr]["cml"].append(level)
+                else:
+                    raise PresenterException(
+                        f"Data error: committee/{committee_nr} is not user committee."
```

Comment:
Make comment more meaningful: f"Data error: user has rights for committee {committee_nr}, but faultily is no member of committee." 
Don't want to write "database corrupt", but the fault should be clear. perhaps you have a better text

Reply:
I don't have a better text. So I used your text.